### ISSUE TITLE
Server does not remember worlds after restart (Fabric 1.19)

### ISSUE THREAD

**Issue by timtijmen2** [IsaiahMC/multiworld#25]
My server doesnt remember created worlds after i restart the server.
The files for the world still exist but are not used.
When i try creating a new world with the same name, notting happens. Not even some sort of chat/console message.

**Comment by jaynotgay** [comment]
Hello! I am also getting this issue. This is a great mod, I hope it can be fixed soon

**Comment by timtijmen2** [comment]
> Hello! I am also getting this issue. This is a great mod, I hope it can be fixed soon
For some reason it deletes the files to register the world sometimes. Its in the config folder, if you recreate the files it *might* work.
Even tho sometimes it just wont..

**Comment by timtijmen2** [comment]
```namespace: prefix
path: worldname
environment: NORMAL
seed: seed```

the file name is  worldname.yml

**Comment by aaron01-dev** [comment]
The fix is: most likely you are using this for singleplayer/lan instance. In a server instance, you'd have to stop and reopen the server for the mw world/configurations to save. In a singleplayer/client instance technically "Save and Quit to Title" is not you closing the server, you'd have to close the game itself for it to save.

tl;dr; you're using a minecraft client not server to open a singleplayer/lan world, you need to close the game.

I believe there's a way to make configs save and reload, but they didn't so you'd have to close and open your mc for to be remembered.
